defaults/main.yml

```diff
@@ -13,5 +13,13 @@ letsencrypt_certbot_verbose: true
 letsencrypt_pause_services: []
 # Force Certificate Reneval
 letsencrypt_force_renew: true
-# Additional certbot arguments
-letsencrypt_certbot_args: []
+# certbot arguments
+letsencrypt_certbot_args: ["certonly",
+   "--text",
+   "-n",
+   "--no-self-upgrade",
+   "-m {{ letsencrypt_email }}",
+   "--domains {{ _letsencrypt_domains | default(letsencrypt_domain) }}",
+   "--agree-tos",
+   "--standalone",
+   "--expand"]
```

Comment:
@robbyoconnor it's just perfect as it is. Later we can figure out which list style is the better one - YAML leaves too much room for opinions 😼